src/Microsoft.Fast.Components.FluentUI/Components/NavMenu/FluentNavMenu.razor

```diff
@@ -2,8 +2,8 @@
 
 @inherits FluentComponentBase
 <div>
-    <CascadingValue Value="this" Name="NavMenu" IsFixed="true">
-        <CascadingValue Value="Expanded" Name="NavMenuExpanded">
+    <CascadingValue Value="this" IsFixed="true">
```

Comment:
No need to name a complex type, and naming it makes it more susceptible to bugs if someone changes one without the other.  Better to have it determined by type instead.